### DISCUSSION TITLE
No input on Wayland

### DISCUSSION THREAD

**Discussion by DarkToaster69** [rustdesk/rustdesk#6378] [General]
### Bug Description

I can't control another PC with Archlinux and Wayland on KDE.

### How to Reproduce

Open RustDesk on your PC and another PC with wayland you want to connect.
Accept the connection on the PC with wayland and try to move the mouse.

### Expected Behavior

The mouse and keyboard to work.

### Operating system(s) on local side and remote side

Arch Linux (Wayland) -> Arch Linux (Wayland)

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

**Comment by beelchester** [comment]
Please ensure if `rustdesk --service` is running as root or not (`ps aux | grep rustdesk`).
If not try `sudo systemctl restart rustdesk`.

**Reply by THEGOLDENPRO** [reply]
I can confirm the service is running as root but I'm still having this issue. (no mouse or keyboard input)

I'm using KDE Plasma 6.0.5 (wayland) on Arch Linux x86_64 with kernel version 6.9.2-arch1-1. (up to date system)

**Reply by piotr-dobrogost** [reply]
@THEGOLDENPRO Have you solved this? I have the same problem on Fedora 42 with KDE 6.3.4. I raised https://github.com/rustdesk/rustdesk/issues/11390 which was converted to https://github.com/rustdesk/rustdesk/discussions/11392
